CSS/works-styles.css

```diff
@@ -0,0 +1,136 @@
+.works {
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  height: 500vh;
```

Comment:
- kindly you should reduce the space between cards. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
## your design
![image](https://user-images.githubusercontent.com/95297251/185511058-b7633769-0a7f-4ecc-bb11-1a284d704a81.png)

## Figma design
![image](https://user-images.githubusercontent.com/95297251/185511635-2279b6ef-f8f4-4818-b34a-affefcd25d9e.png)